kllawesome.py

```diff
@@ -0,0 +1,66 @@
+# imports from third party packages
+from flask import Flask, request, jsonify
+from playhouse.shortcuts import model_to_dict
+
+# imports from inside project
+from model import Config, Keyboard, Category
+
+# config options
+DEBUG = True
```

Comment:
Good idea having a section for this. If it goes in complication it'll be easy to move to a config file too!